src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3585,12 +3585,16 @@
 
     /**
      * Beginning of block comment: '/*'.
-     *
+     * <p>For example:</p>
      * <pre>
-     * +--BLOCK_COMMENT_BEGIN
-     *         |
-     *         +--COMMENT_CONTENT
-     *         +--BLOCK_COMMENT_END
+     * /&#42; -Block Comment Begin
+     * &#42;/
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     * |--BLOCK_COMMENT_BEGIN -&gt; /&#42;
+     * |   |--COMMENT_CONTENT -&gt;   - Block Comment Begin\r\n
```

Comment:
Why there is a space between `-` and `Block Comment Begin\r\n`?

Reply:
@pbludov That is from the output that I get when I run `$ java -jar checkstyle-8.41-all.jar -T Test.java` on my Test.java file

Reply:
Yes. Please replace "- Block Comment Begin" with "Comment content" in the example and in the parsed tree. It will be more relevant.